mobile/lib/modules/backup/services/backup.service.dart

```diff
@@ -41,6 +43,29 @@ class BackupService {
     }
   }
 
+  void saveDuplicatedAssetIdToLocalStorage(String deviceAssetId) {
```

Comment:
this is a very expensive operation to add a single asset ID. a suggest to change the interface and add "batching" by allowing to add a list of asset IDs at once